6.x/base-widgets.md

```diff
@@ -414,6 +414,64 @@ Widget Preview:
 
 <hr>
 
+<a name="livewire"></a>
+### Livewire
+
+Add a Livewire component to a page. If you haven't created your component yet, head to [Livewire documentation](https://livewire.laravel.com/docs/components) and create the component you want to use.
+
+**Important Note**: This widget automatically load the Livewire Assets (scripts and styles) for you. If you are using Livewire in the same page as this widget and is already loading the assets, you can disable the automatic loading by setting the ```livewireAssets``` attribute to ```false```.
+
+```php
+[
+    'type'        => 'livewire',
+    'content'   => 'my-livewire-component',
+    'parameters'      => ['user' => backpack_user(), 'param2' => 'value2'],
+    'livewireAssets' => true, // set false to dont load livewire assets
+]
+```
+
+**Note:** The ```parameters``` attribute will be passed to the component on initialization, and should be present in the `mount($user, $param2)`.
+
+
+##### HelloWord Example:
+
+```php
+use Livewire\Component;
+
+class HelloWorld extends Component
+{
+    public $name;
+
+    public function mount(string $name)
+    {
+        $this->name = $name;
+    }
+
+    public function render()
+    {
+        return view('livewire.hello-world');
+    }
+}
+```
+
+```blade
+<!-- livewire/hello-world.blade.php -->
+<div>
+    Hello {{ $name }}
+</div>
+```
+
+```php
+// add the widget to the page
+Widget::add()->type('livewire')->content('hello-world')->parameters(['name' => 'John Doe'])->wrapperClass('col-md-12 text-center');
```

Comment:
Could we use “component” instead of “content”? Would it be smart or atypical?

Reply:
"content" is on every widget from divs to cards, it seem'd to me that giving the "content" of the "livewire" widget made sense too. I like it more than creating a different key. 

can always revert, or use both as aliases 🙏

Reply:
Ok let’s keep content